ext/configuration.h

```diff
@@ -153,6 +154,7 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(BOOL, DD_TRACE_DEBUG_CURL_OUTPUT, "false", .ini_change = zai_config_system_ini_change)              \
     CONFIG(INT, DD_TRACE_BETA_HIGH_MEMORY_PRESSURE_PERCENT, "80", .ini_change = zai_config_system_ini_change)  \
     CONFIG(BOOL, DD_TRACE_SIDECAR_TRACE_SENDER, "false", .ini_change = zai_config_system_ini_change)           \
+    CONFIG(BOOL, DD_TRACE_BYPASS_AGENT, "false", .ini_change = zai_config_system_ini_change)                   \
```

Comment:
I think there is some existing environment variable that's named differently. Ask Brett and other and see if you can find out.